storage/innobase/trx/trx0trx.cc

```diff
@@ -1560,6 +1560,9 @@ trx_flush_log_if_needed_low(
 
 	switch (srv_flush_log_at_trx_commit) {
 	case 3:
+               /* Write the log and optionally flush it to disk */
```

Comment:
Can you change like below:
```
-       case 3:
        case 2:
                /* Write the log but do not flush it to disk */
                flush = false;
                /* fall through */
        case 1:
+       case 3:
                /* Write the log and optionally flush it to disk */
                log_write_up_to(lsn, flush);

```

Reply:
Thanks for the feedback. I have update code as suggested.